lib/Doctrine/Migrations/Tools/Console/ConsoleLogger.php

```diff
@@ -100,10 +103,12 @@ public function log($level, $message, array $context = []): void
     /**
      * Interpolates context values into the message placeholders.
      *
-     * @param mixed[] $context
+     * @param string|Stringable $message
```

Comment:
I would expect PHPStan to complain about this type not existing in PHP 7, but apparently, it's not an issue… if you know why, I'm interested :slightly_smiling_face:

Reply:
`symfony/polyfill-php80` ships the interface. 🙂